src/olympia/amo/tests/test_cron.py

```diff
@@ -31,7 +31,7 @@ def test_stale_activity_logs_deletion(self, storage_mock):
             ActivityLog.objects.create(action=amo.LOG.THROTTLED.id, user=user),
             # Old activity from action that should always be kept.
             ActivityLog.objects.create(
-                action=amo.LOG.LOG_IN.id, user=user, created=self.days_ago(181)
+                action=amo.LOG.LOG_IN.id, user=user, created=self.days_ago(366)
```

Comment:
And then `constant + 1` here and the other test.

Reply:
[`3ea0d55` (#20229)](https://github.com/mozilla/addons-server/pull/20229/commits/3ea0d55d748cfa225e4f4d532e5751faa0650bca)